cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,69 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import settingPageObject from '../support/pages/settings.pageObject';
+import { url } from 'inspector';
+
+const signInPage = new SignInPageObject();
+const settingsPage = new settingPageObject();
+
 describe('Settings page', () => {
+  let user;
+  
   before(() => {
-
+    cy.task('db:clear');
```

Comment:
you do not need to clear here db, because you have the same method in beforeEach hook